backend/api/v1/endpoints/research.py

```
@@ -33,30 +36,28 @@
+        save_upload_file(file, UPLOAD_DIR, safe_filename)
 
-        # ✅ Save paper metadata to PostgreSQL
+        # ✅ Save metadata to DB
         new_paper = ResearchPaper(
             title=title,
             author=author,
             research_field=research_field,
-            file_path=str(file_path),  # Store path as string
-            uploader_id=current_user.id
+            file_path=safe_filename,  # Only the filename
+            uploader_id=current_user.id,
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)